### PR TITLE
use oauth2file to populate the oauth2-redirect.html always with a fqdn

### PR DESCRIPTION
**Pull request by abrahammartin**

### CODE REVIEW

File: assets/templatetags/oauth2file.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+from django.contrib.staticfiles.templatetags.staticfiles import static
+from urllib.parse import urlparse, urljoin
+
+
+register = template.Library()
+
+
+@register.simple_tag
+def oauth2file(scheme, host):
+    oauth2fileurl = urlparse(static('iarbackend/oauth2-redirect.html'))
```
Comment (rjw57) [inline assets/templatetags/oauth2file.py]: Add comment describing why this tag exists?

**Comment by abrahammartin** [comment]
individual tests seem to be working, if you delete one or the other, but the two together doesn't seem to work. No idea why this is happening

File: assets/templatetags/oauth2file.py [hunk 2]
```diff
@@ -7,8 +7,9 @@
...
 @register.simple_tag
-def oauth2file(scheme, host):
-    oauth2fileurl = urlparse(static('iarbackend/oauth2-redirect.html'))
+def oauth2file(scheme, host, static_file):
+    """This custom tag always generates a FQDN for the specified static file"""
+    oauth2fileurl = urlparse(static(static_file))
     if oauth2fileurl.scheme == '' and oauth2fileurl.netloc == '':
-        return urljoin(str(scheme)+str(host), static('iarbackend/oauth2-redirect.html'))
-    return static('iarbackend/oauth2-redirect.html')
+        return urljoin(str(scheme)+"://"+str(host), static(static_file))
```
Comment (rjw57) [inline assets/templatetags/oauth2file.py]: Can we not use urlunsplit here?

Comment (abrahammartin) [reply assets/templatetags/oauth2file.py]: done

**Pull request merged**